features/promotion/managing_coupons/coupon_generate_validation.feature

```diff
@@ -8,6 +8,7 @@ Feature: Coupon generate instruction validation
         Given the store operates on a single channel in "France"
         And there is a promotion "Christmas sale"
         And it is coupon based promotion
+        And it is enabled promotion
```

Comment:
Promotion is enabled by default, so there is no point in writing it explicit in every scenario. This steps should be removed.